<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="2,2,2,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_1" stretch="3,30">
      <item>
       <widget class="QLabel" name="exp_Label">
        <property name="font">
         <font>
          <family>Aharoni</family>
          <pointsize>36</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>表达式</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="get_exp_textEdit">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>36</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,2,29">
      <item>
       <widget class="QLabel" name="treated_label">
        <property name="font">
         <font>
          <family>Aharoni</family>
          <pointsize>36</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>指示</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="show_pretreated_Label">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>36</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,2,29">
      <item>
       <widget class="QLabel" name="result_label">
        <property name="font">
         <font>
          <family>Aharoni</family>
          <pointsize>36</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>结果</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="show_result_Label">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>36</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,3,2">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cal_pushButton">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="text">
         <string>计算</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>887</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>使用说明</string>
    </property>
    <addaction name="action_use"/>
    <addaction name="action_fileIO"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>其他</string>
    </property>
    <addaction name="actionQT"/>
    <addaction name="action_log"/>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>文件操作</string>
    </property>
    <addaction name="action_edit"/>
    <addaction name="action_output"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="action_use">
   <property name="text">
    <string>支持的运算</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_log">
   <property name="text">
    <string>工作日志</string>
   </property>
  </action>
  <action name="action_fileIO">
   <property name="text">
    <string>文件输入输出</string>
   </property>
  </action>
  <action name="action_edit">
   <property name="text">
    <string>编辑文件</string>
   </property>
  </action>
  <action name="action_output">
   <property name="text">
    <string>输出结果到文件尾</string>
   </property>
  </action>
  <action name="actionQT">
   <property name="text">
    <string>QT版本</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
